code-name-taran/src/com/spaghettisoft/component/scrabble/TurnMakeWord.java

```diff
@@ -0,0 +1,185 @@
+package com.spaghettisoft.component.scrabble;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TurnMakeWord {
+
+	Scanner scanner;
+	String word;
+	Stave stave;
+
+	TurnMakeWord(Scanner scanner) {
+		this.scanner = scanner;
+		word = "";
+		stave = new Stave('h', 0, 0);
+	}
+
+	public void placeWord(Player player, Board board, Bag bag) {
+
+		askPlayerToInsertValidWordAndCoordinates(player,board);
+		int wordPoints = drawWord(player, board, bag);
+		player.increasePoints(wordPoints);
+		setMultipliersTo1(word, stave.getRow(), stave.getCol(), stave.getCh(), board);
+
+		System.out.println(player.getName() + " has " + player.getPoints() + " points.\n");
+	}
+
+	private void setMultipliersTo1(String word, int row, int col, char orientation, Board board) {

```

Comment:
It is strange to use digits inside method names. I think it will be better if you replace "1" with "one".